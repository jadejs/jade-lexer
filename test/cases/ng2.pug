div(
  *star="*"
  (baz)="cow"
  [box]="box"
)
