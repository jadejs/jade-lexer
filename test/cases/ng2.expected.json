{"type":"tag","line":1,"col":1,"val":"div"}
{"type":"start-attributes","line":1,"col":4}
{"type":"attribute","line":2,"col":3,"name":"*star","val":"\"*\"","mustEscape":true}
{"type":"attribute","line":3,"col":3,"name":"(baz)","val":"\"cow\"","mustEscape":true}
{"type":"attribute","line":4,"col":3,"name":"[box]","val":"\"box\"","mustEscape":true}
{"type":"end-attributes","line":5,"col":1}
{"type":"newline","line":6,"col":1}
{"type":"eos","line":6,"col":1}